bokun.yaml

```diff
@@ -5952,14 +6271,62 @@ definitions:
         description: 'Only used if hasOpeningHours == true. The default opening hours by weekday. These may be overridden by seasonal opening hours.'
       seasonalOpeningHours:
         type: array
-        description: 'Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
+        description: 'Deprecated. Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
         items:
           $ref: '#/definitions/SeasonalOpeningHours'
+      displaySettings:
+        type: object
+        properties:
+          showPickupPlaces:
+            type: boolean
+            description: 'Specifies if pick up places are shown.'
+          showRouteMap:
+            type: boolean
+          selectRateBasedOnStartTime:
+            type: boolean
+          customFields:
+            type: array
+            items:
+              type: object
+            description: 'Deprecated.'
+      hasBoxes:
+        type: boolean
+        description: 'Not working.'
+      requestDeadline:
+        type: integer
+        format: int64
+        description: 'The number of miliseconds allocated to confirm or reject booking by activity supplier.'
```

Comment:
Is this not a mistake? There's requestDeadlineMinutes/hours/days/weeks  which I think is the correct way of expressing request deadline; and this one should be removed. Also it is important to emphasize that the deadline is against *travel time* meaning that if no confirmation is received by the deadline, the booking is rejected.

Reply:
This one is similar to "bookingCutoff" (difference is that minutes are used there). This property combines all 4 requestDeadline[x] properties and sum them up. And it is working.